data-catalog/overlays/dev/argocd-rb.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: authorization.openshift.io/v1
+kind: RoleBinding
```

Comment:
actually, you shouldn't need this at all, right? Since you already have it in the base above

Reply:
I think he should remove it from the base